internal/btpcli/facade_accounts_directory.go

```diff
@@ -33,7 +33,7 @@ type DirectoryCreateInput struct {
 	Subdomain     *string             `btpcli:"subdomain"`
 	Labels        map[string][]string `btpcli:"labels"`
 	Globalaccount string              `btpcli:"globalAccount"`
-	//DirectoryAdmins string          `btpcli:"directoryAdmins"`
```

Comment:
Did you remove this on purpose?

Reply:
Yes this was the original state and I left it commented as this functionality is inconsistent compared to the subaccount (not possible to specify further admins )and inconsistent concerning the overall handling of assignment of role collections.
This resource should create/modify/delete the entity `directory`. Any role specifics shall be handled by the corresponding role collection assignment functionality